graphql/subcourse/mutations.ts

```diff
@@ -254,6 +265,7 @@ export class MutateSubcourseResolver {
         // Joining the subcourse will automatically remove the pupil from the waitinglist
         await joinSubcourse(subcourse, pupil, true);
         await addGroupAppointmentsParticipant(subcourseId, user.userID);
+        await addParticipant(user, subcourse.conversationId, subcourse.groupChatType);
```

Comment:
We haven't yet migrated existing courses to have a chat, you assume here that `subcourse.conversationId` is always set. Couldn't we wrap this in an `if (subcourse.conversationId)` to make sure this only runs for courses that already have a chat?